chsdi/lib/helpers.py

```diff
@@ -102,15 +102,6 @@ def make_agnostic(path):
         return path
 
 
-def make_api_url(request, agnostic=False):
-    base_path = request.registry.settings['apache_base_path']
-    base_path = '' if base_path == 'main' else '/' + base_path
-    host = request.host + base_path if 'localhost' not in request.host else request.host
-    if agnostic:
-        return ''.join(('//', host))
-    return ''.join((request.scheme, '://', host))
-
-
```

Comment:
Without checking the code I'm just not 100% sure this was really just used for the shortener, but I assume you've checked that.